src/BreathOfTheWild/Enhancements/rules.txt

```diff
@@ -2,7 +2,7 @@
 titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
 name = Enhancements
 path = "The Legend of Zelda: Breath of the Wild/Enhancements"
-description = Allows you to change post-processing effects and texture detail to enhance the picture.||Made by Kiri, Jamie, M&&M, Pig, Skyth, and clarity preset creators.
+description = Allows you to change post-processing effects and texture detail to enhance the picture.||Made by Kiri, Jamie, M&&M, Pig, Skyth, YeMaoWuXin, and clarity preset creators.
```

Comment:
We don't credit each individual clarity preset creator in the description since it'd be kinda too long. You do get credits in the preset's name ofc!

Reply:
Modified on the basis of Survival of the Wild Preset (Warmer).My native language is not English, so some words I don't understand very well.I wanted to describe bright and colorful, which is why I used the word Brightly.Maybe brightly colored would be better?

Reply:
OK now?